scm-ui/ui-components/src/forms/DropDown.tsx

```diff
@@ -33,6 +34,10 @@ type Props = {
   disabled?: boolean;
 };
 
+const FullWidthSelect = styled.select`
+  width: 100%;
+`;
+
```

Comment:
Is it safe to go always fullwidth? Do we break other layouts with this change?

Reply:
The only side effect I've found is that you can click on the whitespace besides the text of a drop box in a column, too. Don't know, if this is a major issue. Layouts do not seem to be effected, because with this the select only fills the surrounding div, and this one is restricted the same way it was before.

Do you have another solution in mind, though?

Reply:
Checked this and found no errors.